src/main/kotlin/nl/info/client/brp/BrpClientService.kt

```diff
@@ -54,22 +62,27 @@ class BrpClientService @Inject constructor(
 
     fun queryPersonen(personenQuery: PersonenQuery): PersonenQueryResponse =
         updateQuery(personenQuery).let {
-            personenApi.personen(it)
+            personenApi.personen(
+                personenQuery = it,
+                purpose = purposeSearch.validatePurpose("brp.doelbinding.zoekmet")
```

Comment:
Better to make a constant out of the string `brp.doelbinding.zoekmet` I would think?

Reply:
There is a constant, but it's not used, so not resolved yet.

Reply:
```suggestion
                purpose = purposeSearch.validatePurpose(ENV_VAR_BRP_DOELBINDING_ZOEKMET)
```